libs/components/core/src/lib/modules/scrollable-host/scrollable-host.service.ts

```diff
@@ -190,6 +204,42 @@ export class SkyScrollableHostService {
     });
   }
 
+  public watchScrollableHostClipPathChanges(
+    elementRef: ElementRef
+  ): Observable<string> {
+    if (!this.#resizeObserverSvc) {
+      throw new Error(
+        'The SkyResizeObserverService is required to use the watchScrollableHostClipPathChanges method.'
+      );
+    }
+
+    return this.watchScrollableHost(elementRef).pipe(
+      switchMap((scrollableHost) => {
+        if (
+          !this.#resizeObserverSvc ||
+          scrollableHost === this.#windowRef.nativeWindow
+        ) {
+          return of('none');
+        }
+
+        return concat([
+          of(undefined),
+          this.#resizeObserverSvc.observe({ nativeElement: scrollableHost }),
```

Comment:
Recalculate one time when there is a new scrollable host plus whenever the scrollable host changes shape.